util/src/main/java/io/camunda/zeebe/util/startup/StartupProcess.java

```diff
@@ -192,7 +191,6 @@ private void shutdownSynchronized(
       final ActorFuture<CONTEXT> resultFuture) {
     logger.debug("Shutdown was called with context: " + context);
```

Comment:
:x: Sorry I missed this earlier - use the logger's string interpolation and not string concatenation to avoid extra work if the log level is not debug, e.g.

```java
logger.debug("Shutdown was called with context: {}", context);
```